portable/NetworkInterface/ThirdParty/MSP432/NetworkInterface.c

```diff
@@ -856,20 +855,22 @@ bool isEMACLinkUp()
 {
     uint8_t uci8PHYAddr = 0; /* refers to the internal PHY */
     uint16_t check;
+    BaseType_t returnValue;
+    returnValue = pdTRUE;
```

Comment:
maintainability: it's better to initialize return values to fail/error rather than success, e.g.:

```c
BaseType_t returnValue = pdFALSE;

check = EMACPHYRead( EMAC0_BASE, uci8PHYAddr, EPHY_BMSR );

if( ( ( check & EPHY_BMSR_LINKSTAT ) != 0 ) && !( check & EPHY_BMSR_RFAULT ) )
{
    returnValue = pdTRUE; /* link is up */
}

return returnValue; /* return link status */

Reply:
I agree, and the code has been updated.